packages.yml

```diff
@@ -3,5 +3,9 @@ packages:
   - package: fishtown-analytics/dbt_utils
     version: [">=0.6.0","<0.7.0"] 
 
+  - package: fishtown-analytics/spark_utils
+    version: [">=0.2.0", "<0.3.0"]
+
   - git: https://github.com/fivetran/dbt_fivetran_utils.git
-    warn-unpinned: false  
\ No newline at end of file
+    warn-unpinned: false  
+    revision: postgres_percentile
```

Comment:
Remember to remove the dbt-utils and old fivetran-utils dependencies before merging. You will keep spark utils as is.